tajo-plan/src/main/java/org/apache/tajo/plan/logical/TableSubQueryNode.java

```diff
@@ -108,14 +111,19 @@ public boolean hasTargets() {
   }
 
   @Override
-  public void setTargets(Target[] targets) {
+  public void setTargets(List<Target> targets) {
     this.targets = targets;
     setOutSchema(PlannerUtil.targetToSchema(targets));
   }
 
   @Override
-  public Target[] getTargets() {
-    return targets;
+  public List<Target> getTargets() {
+    if (hasTargets()) {
+      return targets;
```

Comment:
I make JIRA ticket. https://issues.apache.org/jira/browse/TAJO-1976 . Thanks for your reply. Anything else to commit this PR?